rootfs/scheduler/resources/ingress.py

```diff
@@ -23,6 +23,8 @@ def get(self, name=None, **kwargs):
         return response
 
     def create(self, ingress, namespace, hostname):
+        # Ingress API will be deprecated in 1.20 to use networking.k8s.io/v1beta1
+        # https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/
         url = "/apis/extensions/v1beta1/namespaces/%s/ingresses" % namespace
```

Comment:
Since the new endpoint exists since 1.14, should this check if `self.version() >= parse("1.14.0")` and then use `networking.k8s.io/v1beta1` instead of `extensions/v1beta1` in the url path?

Reply:
I think it makes sense to extract a property method `api_version` here as well, since it is currently hardcoded in multiple places.

Reply:
Done in d8e65a9. Thank for pointing at this.